docs/pages/reference/cli.mdx

```diff
@@ -327,6 +327,106 @@ $ tsh play --format=pty 1fe153d1-ce8b-4ef4-9908-6539457ba4ad.tar
+Database connections can be made with certificates through database clients or tunnel mode
+that does not require authentication.
+
+```code
+$ tsh proxy db [<flags>] <db>
+```
+
+#### Flags
+
+| Name | Default Value(s) | Allowed Value(s) | Description |
+| - | - | - | - |
+| `--cert-file` | none | Certificate file| Certificate file for proxy client TLS configuration |
+| `--key-file` | none | Key file| Key file for proxy client TLS configuration |
+| `tunnel` | none | Enable tunnel mode | Open authenticated tunnel using database's client certificate so clients don't need to authenticate |
+| `--db-user` | none | Database user| Optional database user to log in as. |
+| `--db-name` | none | Database name| Optional database name to log in to |
+
+
+#### [Global Flags](#tsh-global-flags)
+
+These flags are available for all commands `--login, --proxy, --user, --ttl, --identity, --cert-format, --insecure, --auth, --skip-version-check, --debug, --jumphost, --format`.
+Run `tsh help <subcommand>` or see the [Global Flags Section](#tsh-global-flags)
+
+#### Examples
+```code
+$ tsh proxy db <db>
+
+# Proxy a connection to mysql db on local port 10700
+$ tsh proxy db --port 10700  mysql-db
+# Started DB proxy on 127.0.0.1:10700
+
+# Use following credentials to connect to the mysql-db proxy:
+#  ca_file=/Users/jeff/.tsh/keys/teleport.example.com/cas/teleport.example.com.pem
+#  cert_file=/Users/jeff/.tsh/keys/teleport.example.com/jeff-db/tele1c/mysql-db-x509.pem
+#  key_file=/Users/jeff/.tsh/keys/teleport.example.com/jeff
+
+# Proxy a connection to mysql db with no credentials required
+$ tsh proxy db --tunnel mysql-db
+Started authenticated tunnel for the MySQL database "mysql-db" in cluster "teleport.example.com" on 127.0.0.1:49415.
+
+Use the following command to connect to the database:
+  $ mysql --port 49415 --host localhost --protocol TCP
+```
+
+### tsh proxy ssh
+
+Start local TLS proxy for `ssh` connections when using Teleport in single-port mode.
+This is typically used as part of the SSH client configuration to use `ssh` as a client 
+through Teleport.  See [OpenSSH Guide](../server-access/guides/openssh.mdx)
+on configuring OpenSSH servers and clients.  The `tsh config` output will include `tsh proxy ssh`
+within the `ProxyCommand`.
+
+```code
+$ tsh proxy ssh [<flags>] <[user@]host>
+```
+
+#### Arguments
+- `<[user@]host>` Remote hostname and the login to use
+
+#### Flags
+
+| Name | Default Value(s) | Allowed Value(s) | Description |
+| - | - | - | - |
+| `--cluster` | none | Teleport Cluster | Specify the Teleport cluster to connect |
+
+#### [Global Flags](#tsh-global-flags)
+
+These flags are available for all commands `--login, --proxy, --user, --ttl, --identity, --cert-format, --insecure, --auth, --skip-version-check, --debug, --jumphost, --format`.
+Run `tsh help <subcommand>` or see the [Global Flags Section](#tsh-global-flags)
+
+#### Examples
+```code
```

Comment:
Is there ever a reason to use this command outside of the `tsh config` output? That might be good to include here.